src/styles/styles.ts

```diff
@@ -480,7 +480,9 @@ const styles = (theme: ThemeDefault) =>
             borderRadius: variables.buttonBorderRadius,
             minHeight: variables.componentSizeLarge,
             justifyContent: 'center',
+            alignItems: 'center',
```

Comment:
i think this change caused this [issue](https://github.com/Expensify/App/issues/30400)